sklearn/tests/test_common.py

```diff
@@ -309,6 +311,8 @@ def _estimators_that_predict_in_fit():
     "estimator", column_name_estimators, ids=_get_check_estimator_ids
 )
 def test_pandas_column_name_consistency(estimator):
+    if isinstance(estimator, ColumnTransformer):
```

Comment:
this and `test_check_param_validation` are being moved to `estimator_checks` and therefore will be able to be skipped on the estimator tag, for now they need to be skiped hardcoded here. The PR moving these tests will fix this issue as well.